README.md

```diff
@@ -82,17 +82,17 @@ e.g.`chat.kickUser(user.id)`, `message.editText("edited")`.
 
 ### Webhook support
 **canoe** also provides support for obtaining messages from Telegram by setting a webhook.
-Full example may be found [here](https://github.com/augustjune/canoe/blob/master/examples/src/main/scala/samples/WebhookGreetings.scala).
+Full example may be found [here](https://github.c?om/augustjune/canoe/blob/master/examples/src/main/scala/samples/Webhook.scala).
```

Comment:
github's domain should probably stay the same ;)

Reply:
:man_facepalming: I have just pushed the fix to the typo :+1: